src/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
- [ ] You have done a great job setting up webpack, everything works perfectly well 👏🏻 👏🏻 👏🏻 
The only requirement that you still did not meet is that your HTML should look like in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png). No styling is required on this stage, just make sure to make a similar layout:

Your layout  | Wireframe layout
------------- | -------------
<img width="1435" alt="Screenshot 2022-04-12 at 10 32 32" src="https://user-images.githubusercontent.com/77038610/162895545-bc93029e-a081-4442-b442-96453031bc4a.png"> | <img width="614" alt="Screenshot 2022-04-12 at 10 32 18" src="https://user-images.githubusercontent.com/77038610/162895501-8c5a182a-711a-441e-b34d-e41668522b88.png">

Reply:
@ShoiraTa, Thank you for your feedback, I'll work on it.